ultimatejobweb/boostraped_website_draft/sign_in.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Why not adding login page? you will have 
Home, Signup, Login, it is a progress

Reply:
Hey,
just updated all the changes.
There is a login page, i just named it "sign in".
Right now the "sign up" option works and saves the details of the new user in the users DB using django form.
once a user signes up- he is redirected to his personal area, and from there he can go and see his job offers.